src/util/request-manager.js

```diff
@@ -8,12 +8,13 @@ import * as network from './network.js';
 import map from '../util/map.js';
 
 import typeof * as RequestModuleT from 'request';
+import request from 'request'
```

Comment:
nit: [semicolon](https://www.youtube.com/watch?v=M94ii6MVilw) (but don't fix it, cf below)